runcodeanalysis.ps1

```diff
@@ -0,0 +1,93 @@
+[CmdletBinding(DefaultParameterSetName='RegularBuild')]
+param (
+    [ValidateSet("debug", "release")]
+    [string]$Configuration = 'debug',
+    [int]$BuildNumber,
+    [switch]$SkipRestore,
+    [switch]$CleanCache,
+    [string]$SimpleVersion = '1.0.0',
+    [string]$SemanticVersion = '1.0.0-zlocal',
+    [string]$Branch,
+    [string]$CommitSHA,
+    [string]$BuildBranch = '8d2ae35f508cff4fa1fb4e758e61195938e315b5',
+    [string]$FxCopDirectory,
+    [string]$FxCopProject,
+    [string]$FxCopRuleSet,
+    [string]$FxCopOutputDirectory
+)
```

Comment:
A lot of duplication. Alternatively, I could integrate with `build.ps1` which will require adding alot of new parameters.